sdk/dnsresolver/Microsoft.Azure.Management.DnsResolver/src/Microsoft.Azure.Management.DnsResolver.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+	<!-- Please do not move/edit code below this line -->
```

Comment:
above?

Reply:
Checked both privatedns and other services and they all have same doc like this. 
![image](https://user-images.githubusercontent.com/12420143/106942814-78780500-66d9-11eb-93b4-bd84c6768e8b.png)